### ISSUE TITLE
Ensure upload Job is managed correct if process error occurs

### ISSUE THREAD

**Issue by palmerj** [linz/linz_bde_uploader#12]
If an error during any SQL task run within dataset_load_end_sql configuration the job is status is still set as active and the working schema is not cleaned up.

There is some flaw in either in LINZ::BdeDatabase::finishJob or the surrounding exception management code in LINZ::BdeUpload

**Comment by palmerj** [comment]
Fixed.

**Comment by billgeo** [comment]
Note: This is currently holding up migration from xxxgeo to xxxassgeo servers.

**Comment by palmerj** [comment]
For now can't the job just be reset? e.g 

```sql
UPDATE bde_control.upload
SET status = 'E'
WHERE status = 'A'
```

?

**Comment by palmerj** [comment]
This has been resolved in version 2.0.2 via 9466606
